rdrf/rdrf/services/rest/views/proms_api.py

```diff
@@ -34,6 +35,17 @@ def multicde(cde_model):
     return datatype == "range" and cde_model.allow_multiple
 
 
+def get_registry(code):
+    query_cache = caches["queries"]
+    key = f"Registry_{code}"
+    if key in query_cache:
+        return query_cache.get(key)
+    else:
```

Comment:
can't we abstract this into a helper function in utils

ie
def get_rdrf_model(model):
     key = get_key(model)
     if key in cache etc